everviz/plugins/_crossplot/_callback/_crossplot_callback.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
```

Comment:
The same question regarding the `_` in the name for this file and for the `_callback` module and also for the `_crossplot` module.